buổi-9/bai4/bai-4.js

```diff
@@ -0,0 +1,27 @@
+function maxNumber(){
```

Comment:
tên hàm phải là động từ
```suggestion
function getMaxNumber(a, b, c) {
```